architecture/additional_concepts/storage.adoc

```diff
@@ -0,0 +1,178 @@
+
+== Lifecycle of a volume and claim
+
+PVs are resources in the cluster.  PVC are requests for those resources and also act as claim checks to the resource.  The interaction between PVs and PVCs follows this lifecycle:
+
+=== Provisioning
+
+A cluster administrator creates some number of PVs. They carry the details of the real storage that is available for use by cluster users.  They exist in the API and are available for consumption.
+
+=== Binding
+
+A user creates a `PersistentVolumeClaim` with a specific amount of storage requested and with certain access modes.  A control loop in the master watches for new PVCs, finds a matching PV (if possible), and binds them together.  The user will always get at least what they asked for, but the volume may be in excess of what was requested.
+
+Claims will remain unbound indefinitely if a matching volume does not exist.  Claims will be bound as matching volumes become available.  For example, a cluster provisioned with many 50Gi volumes would not match a PVC requesting 100Gi.  The PVC can be bound when a 100Gi PV is added to the cluster.
+
+=== Using
+
+Pods use claims as volumes. The cluster inspects the claim to find the bound volume and mounts that volume for a pod.  For those volumes that support multiple access modes, the user specifies which mode desired when using their claim as a volume in a pod.
+
+Once a user has a claim and that claim is bound, the bound PV belongs to the user for as long as she needs it. Users schedule Pods and access their their claimed PVs by including a persistentVolumeClaim in their Pod's volumes block. See link:#claims-as-volumes[below] for syntax details.
+
+=== Releasing
+
+When a user is done with their volume, they can delete the PVC objects from the API which allows reclamation of the resource.  The volume is considered "released" when the claim is deleted, but it is not yet available for another claim.  The previous claimant's data remains on the volume which must be handled according to policy.
+
+=== Reclaiming
+
+The reclaim policy of a `PersistentVolume` tells the cluster what to do with the volume after it's released.  Currently, volumes can either be Retained or Recycled.  Retention allows for manual reclamation of the resource.  For those volume plugins that support it, recycling performs a basic scrub ("rm -rf /thevolume/*") on the volume and makes it available again for a new claim.
+
+== Persistent Volumes
+
+== Types of Persistent Volumes
+
+OpenShift currently supports the following `PersistentVolume` plugins:
+
+* NFS
+* GCEPersistentDisk
+* AWSElasticBlockStore
+* Glusterfs
+* iSCSI
+* RBD (Ceph Block Device)
+* HostPath (single node testing only)
```

Comment:
Fair to say something like what's below or should they be gone altogether?
  

More plugins are available but still under development: 

* Glusterfs
* iSCSI
* RBD (Ceph Block Device)